src/inc/vnode.h

```diff
@@ -22,6 +22,12 @@ extern "C" {
 #include "trpc.h"
 #include "twal.h"
 
+typedef struct {
+  int32_t submitReqSucNum;
+  int32_t submitRowNum;
+  int32_t submitRowSucNum;
```

Comment:
error num is defined as int64_t, these are defined as int32_t?

Reply:
Good point. I can change this counter to int64_t to mitigate overflow problems.